hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestClientScannerTimeouts.java

```diff
@@ -174,18 +178,27 @@ public void testRetryOutOfOrderScannerNextExceptionAsync() throws IOException {
    * timed out next() call and mess up the test.
    */
   @Test
-  public void testNormalScanTimeoutOnNext() throws IOException {
+  public void testNormalScanTimeoutOnNextWithLegacyMode() throws IOException {
```

Comment:
Can you call this something other than legacy mode? The word "legacy" is always relative, by definition.

Reply:
Removed as part of the parameterized change